### ISSUE TITLE
Encountered some error when evaluating 

### ISSUE THREAD

**Issue by Ianpengg** [nachiket92/PGP#14]
Hello thank you for the great work
I have some problems when I try to evaluate the metrics.
I use the CUDA11.0 and the pytorch 1.7.1
`   
conda install pytorch==1.7.1 torchvision==0.8.2 torchaudio==0.7.2 cudatoolkit=11.0 -c pytorch
`

And I download the trained weights  
![image](https://user-images.githubusercontent.com/88025855/162127895-2e0e7dab-1b83-46b6-a36a-14a6db834a59.png)

I load the weight as 
`-w /media/ee904/Data_stored/PGP/PGP_lr-scheduler/archive/data.pkl`
then the error pop up I think it's about the weight format..
Can you help me? Or should I need to use the older version of CUDA?

![image](https://user-images.githubusercontent.com/88025855/162127332-ea240856-7fb3-4b7c-9e0a-1e274dd6c13f.png)

**Comment by Ianpengg** [comment]
I think there could be different versions between train and test

**Comment by bkwmartens** [comment]
Loading the .tar file itself as weights should solve your problem.

**Comment by nachiket92** [comment]
Yes, please directly pass the .tar file as the weights.
`-w /media/ee904/Data_stored/PGP/PGP_lr-scheduler.tar`
